src/components/MetricCard.js

```diff
@@ -5,53 +5,76 @@ import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 import CircularProgress from '@material-ui/core/CircularProgress';
 import Paper from '@material-ui/core/Paper';
-import Typography from '@material-ui/core/Typography';
-import Box from '@material-ui/core/Box';
 
 import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components/macro';
+
+const TitleBoxStyle = styled.div`
+  border-top: ${props => (props.mediaMax960 ? '1px solid' : '0')};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  ${props =>
+    (props.mediaMax960 || props.bottomPanelOpen) &&
+    css`
+      height: 58px;
+    `}
+  ${props =>
+    !props.mediaMax960 &&
+    css`
+      font-weight: bold;
+    `};
+  font-size: ${props => (props.mediaMax960 ? '20px' : props.bottomPanelOpen ? '15px' : '11px')};
```

Comment:
In this case, you're opening up if-statements three times. Can you refactor it so that there's just one ternary?